server/indexers/transactions.ts

```diff
@@ -12,25 +17,42 @@ export default async function transactions(connection: PoolClient, provider: eth
   async function process() {
     const bids = await findBidsWithMissingTransactions.run({ limit: 5 }, connection);
 
-    log.debug(`Rows: ${bids.length}`);
-
     if (bids.length === 0) {
+      const latestBids = await findWalletsInLatestAuction.run(undefined, connection);
+
+      log.debug(`Rows: ${bids.length} ${latestBids.length}`);
+
+      const latestBalances = await Promise.all(
+        latestBids.map((row) => provider.getBalance(row.walletAddress)),
+      );
+
+      for (const [index, row] of latestBids.entries()) {
+        await updateWalletsValuesInLatestAuction.run(
+          {
+            tx: row.tx,
+            index: row.index,
+            walletBalance: latestBalances[index].toString() || null,
+          },
+          connection,
+        );
+      }
```

Comment:
This shouldn't be here. In the current form it basically says "only process latest wallet balances when we processed all transactions".

Instead, we should probably do something like:

```
const bids = …
if (bids.length) {
  // process bids
}

const latestBids = …
if (latestBids.length) {
 // process latest bids
}
```

Overall it feels like it might need to be a separate indexer altogether — although it's very related to what this one is doing, it needs to run in parallel to the tx metadata backfilling